### DISCUSSION TITLE
Unable to parse yaml when building

### DISCUSSION THREAD

**Discussion by paulwongx** [vercel/turborepo#5865] [Help] [answered]
When building the app, I keep getting this error when running `yarn workspace <name_of_my_app> build` in my monorepo. 
I'm running `turbo: "^1.10.12"`

Which yaml file should I be looking into? It's confusing because I have no clue which file it's referring to
```
WARNING  Issues occurred when constructing package graph. 
Turbo will function, but some features may not be available:  1 error occurred:
--
23:55:40.259 | * unable to parse yaml: mapping values are not allowed in this context at line 24 column 15
```

**Comment by anthonyshew** [comment, marked as answer]
If you're still seeing this issue using the latest version of `turbo`, please open an issue with a reproduction. 🙏
